clang/runtime/dpct-rt/include/dpct/blas_utils.hpp

```diff
@@ -224,10 +224,34 @@ using wrapper_float_inout =
 using wrapper_float_in =
     parameter_wrapper_t<float, float, parameter_inout_prop::in>;
 
+enum class math_mode : int {
+  _default,
```

Comment:
what is the specific info for default mode?

Reply:
cublasMath_t has value CUBLAS_DEFAULT_MATH.
I want to use math_mode::_default to migrate it.
While the compute_type::_default can be removed.